test/smoke/src/application.ts

```diff
@@ -62,8 +62,15 @@ export class Application {
 
 	async start(): Promise<any> {
 		await this._start();
-		await this.code.waitForElement('.explorer-folders-view');
+		//{{SQL CARBON EDIT}}
+		await this.code.waitForElement('.object-explorer-view');
+
+		//Original
+		/*
+		await this.code.waitForElement('.explorer-folders-view');	 		await this.code.waitForElement('.object-explorer-view');
```

Comment:
looks like the line breaks got messed up here

Reply:
fixed